files/ja/learn/performance/multimedia/index.md

```diff
@@ -12,7 +12,7 @@ This change means that the aspect ratio of the image is calculated by the browse
 
 In the olden days of web development, it was always seen as a good practice to add `width` and `height` attributes to your HTML {{htmlelement("img")}} elements, so that when browsers first loaded the page, they could put a correctly-sized placeholder box in the layout for each image to appear in when it finally loads.
 
-![Two screenshots the first without an image but with space reserved, the second showing the image loaded into the reserved space.](https://mdn.mozillademos.org/files/16945/ar-guide.jpg)
+![Two screenshots the first without an image but with space reserved, the second showing the image loaded into the reserved space.](/en-US/docs/learn/performance/multimedia/ar-guide.jpg)
```

Comment:
We could just remove the prefix as the platform would fall back to en-US locale.
```suggestion
![Two screenshots the first without an image but with space reserved, the second showing the image loaded into the reserved space.](ar-guide.jpg)
```

Reply:
Thank you for reviewing. I misunderstood that there is something disturbs relative URL auto linking system.